### ISSUE TITLE
Provide examples on how to view what is currently configured

### ISSUE THREAD

**Issue by odellcraig** [aws/karpenter-provider-aws#5543]
### Description

Is it possible to show/list/view what is currently configured on Karpenter? For example, I followed all your getting started guides and I have Karpenter deployed to an EKS cluster and I was messing around with different NodePools by re-applying a cat-ed yaml file like this:
```
cat <<EOF | envsubst | kubectl apply -f -
apiVersion: karpenter.sh/v1beta1
kind: NodePool
metadata:
  name: default
```
Turns out I had two NodePools, but no visibility into that fact other than watching the logs carefully. I was able to do a kubectl delete to remove one NodePool, but I'd love to have a way to list or describe these things. Sorry if there is a way already that's documented. I looked through all the documentation and did a lot of searching and didn't see anything. 

In short is there a way to do something like
```
kubectl <<describe-node-pools>>
  > 1) default
  > 2) spot
...
```

**How can the docs be improved?**

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by jmdeal** [comment]
You can describe nodepools and any other resource from Karpenter (EC2NodeClasses, NodeClaims, alpha resources) in the same way as any other kubernetes resource, e.g.
```console
$ kubectl get nodepools
$ kubectl describe nodepool <nodepool name>
$ # etc
```
I don't believe this is called out in the docs explicitly, but this is standard for kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/). I don't think this would be a bad callout to add in the Getting Started Guide if you'd be interested in contributing, though I could see the argument that this is outside of the scope of Karpenter's docs.

**Comment by odellcraig** [comment]
Thanks for getting back to me. I figured it was something simple like that.
I could go either way on the documentation update. I'll take a look at the
contribution guide.

- BR

**Comment by MrBeldum** [comment]
Happy to take this — will open a docs PR adding `kubectl get/describe` examples for NodePools, EC2NodeClasses, and NodeClaims to the Getting Started guide.
